plasmapy/utils/decorators/tests/test_validators.py

```diff
@@ -29,40 +37,22 @@ class :class:`~plasmapy.utils.decorators.validators.ValidateQuantities`.
     )  # type: Dict[str, Any]
     check_defaults = {**unit_check_defaults, **value_check_defaults}
 
-    @staticmethod
-    def foo(x):
-        return x
-
-    @staticmethod
-    def foo_anno(x: u.cm):
-        return x
-
     def test_inheritance(self):
         assert issubclass(ValidateQuantities, CheckUnits)
         assert issubclass(ValidateQuantities, CheckValues)
 
-    def test_vq_method__get_validations(self):
-        # method must exist
-        assert hasattr(ValidateQuantities, "validations")
-        assert hasattr(ValidateQuantities, "_get_validations")
-
-        # setup default validations
-        default_validations = {
-            **self.check_defaults.copy(),
-            "units": [self.check_defaults["units"]],
-        }
-
-        # setup test cases
-        # 'setup' = arguments for `_get_validations`
-        # 'output' = expected return from `_get_validations`
-        # 'raises' = if `_get_validations` raises an Exception
-        # 'warns' = if `_get_validations` issues a warning
-        #
-        _cases = [
+    # set up test cases
```

Comment:
```suggestion
    @pytest.mark.parametrize(
        "attr", ["validations", "_get_validations", "_validate_quantity"]
    )
    def test_attr_existence(self, attr):
        assert hasattr(ValidateQuantities, attr)

    # set up test cases
```

Many of the test methods are doing too many things at once, so it would be good to pull standalone tests where we can.  Testing attribute existence is a good example of this.